opentracing-spring-web-handler-interceptor/src/main/java/io/opentracing/contrib/spring/web/servlet/TracingHandlerInterceptor.java

```diff
@@ -0,0 +1,102 @@
+package io.opentracing.contrib.spring.web.servlet;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.springframework.web.servlet.handler.HandlerInterceptorAdapter;
+
+import io.opentracing.Span;
+import io.opentracing.SpanContext;
+import io.opentracing.Tracer;
+import io.opentracing.contrib.web.servlet.filter.HttpServletRequestExtractAdapter;
+import io.opentracing.contrib.web.servlet.filter.TracingFilter;
+import io.opentracing.propagation.Format;
+import io.opentracing.tag.Tags;
+
+/**
+ * Tracing handler interceptor for spring web. It creates a new span for an incoming request.
+ * Created span can be accessed via {@link HttpServletRequest#getAttribute(String)} with name
+ * {@link TracingHandlerInterceptor#ACTIVE_SPAN}.
```

Comment:
> Personally, I would do this differently. I would log the following fields only:
> event: "exception"
> exception: throwable
> and let the tracer implementation deal with exact representation of the exception / stack trace.

I agree this depends on tracer impl, probably users using some specific tracer would use different  decorator. As this is a "standard"  decorator, I wanted to add the most common tag/event to represent an error in the system.